src/ansys/fluent/core/utils/deprecate_args.py

```diff
@@ -2,22 +2,27 @@
 
 import functools
 import logging
+from typing import List
 import warnings
 
-from ansys.fluent.core.launcher.pyfluent_enums import Dimension, FluentEnum
 from ansys.fluent.core.warnings import PyFluentDeprecationWarning
 
 logger = logging.getLogger("pyfluent.general")
 
 
 def deprecate_argument(
-    old_arg, new_arg, converter, deprecation_class=PyFluentDeprecationWarning
+    old_arg,
+    new_arg,
+    converter,
+    deprecation_class=PyFluentDeprecationWarning,
 ):
     """Warns that the argument provided is deprecated and automatically replaces the
     deprecated argument with the appropriate new argument."""
 
     def _str_repr(var):
         """Converts a string or FluentEnum variable to quoted string representation."""
+        from ansys.fluent.core.launcher.pyfluent_enums import Dimension, FluentEnum
```

Comment:
Please can we have a solution without mentioning these concrete types. Provide a way to use those enums generically if necessary.

Reply:
Done, Thank you.